src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -154,6 +155,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('fragment_renderer.xml');
         $loader->load('error_catcher.xml');
 
+        if (interface_exists(PsrEventDispatcherInterface::class)) {
```

Comment:
Do we actually need the interface to exist for creating the alias? 🤔

Reply:
We can but the error will be cryptic and runtime, while the missing alias will lead to a compile time error.